presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix a correctness bug that would occur if two tables with mismatching but compatible bucket counts are joined whiles filters are applied to one of the tables.
```

Comment:
@mayankgarg1990 Does this reflect your change?

Reply:
This was not a correctness bug since the functionality was disabled if we hit the condition. So essentially by fixing the issue, we have enabled a performance optimization to kick in

Reply:
@mayankgarg1990 updated to `Fix an issue that would disable a performance optimization ...`

Reply:
`while filters are applied` is too open since this happens only for the `$bucket` filter. 

```
Fix an issue that would disable a performance optimization that would occur if two tables with mismatching but compatible bucket counts are joined and one of the table has a `$bucket` filter.
```